app/models/chargeback_container_project.rb

```diff
@@ -62,7 +62,7 @@ def self.get_keys_and_extra_fields(perf, ts_key)
     project = @data_index.fetch_path(:container_project, :by_group_id, perf.resource_id)
     key = "#{project.id}_#{ts_key}"
 
-    [key, {"project_name"  => project.name}]
+    [key, {"project_name"  => project.old_ems_id.nil? ? project.name : "#{project.name} (Deleted)"}]
```

Comment:
I'm wondering if it makes more sense to have a method on the project instead of relying on the old_ems_id here.  The reason being that if we change what it means under the covers for a project to be deleted (say we rely on deleted_on, instead), then you have to know to change it here.  I was thinking leverage the existing `.archived?` method, which *should* be the same thing.  VMs would also appear the same way in the future.